packages/compass-connections-navigation/src/item-actions.ts

```diff
@@ -84,23 +93,19 @@ export const connectedConnectionItemActions = ({
       icon: 'InfoWithCircle',
       label: 'Show connection info',
     },
-    {
-      action: 'copy-connection-string',
-      icon: 'Copy',
-      label: 'Copy connection string',
-    },
-    {
-      action: 'connection-toggle-favorite',
-      icon: 'Favorite',
-      label: isFavorite ? 'Unfavorite' : 'Favorite',
-    },
     {
       action: 'connection-disconnect',
       icon: 'Disconnect',
       label: 'Disconnect',
       variant: 'destructive',
-    }
-  );
+    },
+    { separator: true },
+    ...connectionManagementActions,
+  ];
+
+  if (isReadOnly) {
+    actions.splice(0, 1);
```

Comment:
What's being removed here?

Reply:
sorry for the overlooking here - i added [a comment](https://github.com/mongodb-js/compass/pull/5877/commits/cd65815340e0839354c9cff6486fcc10af3e6b9c) describing what is getting removed and why.